OrbitClientGgp/ClientGgp.cpp

```diff
@@ -79,9 +78,107 @@ bool ClientGgp::StopCapture() {
   return capture_client_->StopCapture();
 }
 
-void ClientGgp::InitCapture() {
+std::shared_ptr<Process> ClientGgp::GetOrbitProcessByPid(int32_t pid) {
+  // We retrieve the information of the process to later get the module corresponding to its binary
+  ErrorMessageOr<std::vector<ProcessInfo>> result_process_infos = process_client_->GetProcessList();
+  if (result_process_infos.has_error()) {
+    return nullptr;
+  }
+  const std::vector<ProcessInfo>& process_infos = result_process_infos.value();
+  LOG("List of processes:");
+  for (const ProcessInfo& info : process_infos) {
+    LOG("pid:%d, name:%s, path:%s, is64:%d", info.pid(), info.name(), info.full_path(),
```

Comment:
Just as a remark: This seems to be a lot of logging. Keep as is if you think it makes sense.

Reply:
At the beginning I added this LOG with the idea of removing it later once the code was working but I realized it could be quite helpful and informative in case something is wrong so I decided to keep it in the end. E.g. the selected process is not found; could be that they provided the wrong pid so they get aware of the reason by checking the logs; could be the process is not running at all, etc